synapse/tools/axon/load.py

```diff
@@ -0,0 +1,88 @@
+import tarfile
+
+import synapse.exc as s_exc
+import synapse.common as s_common
+import synapse.telepath as s_telepath
+
+import synapse.lib.cmd as s_cmd
+import synapse.lib.output as s_output
+
+descr = '''
+Load blobs into a Synapse Axon.
+'''
+
+DEFAULT_CHUNK_SIZE = s_const.megabyte * 16
```

Comment:
```suggestion
DEFAULT_CHUNK_SIZE = s_const.mebibyte * 16
```
lets match the axon default chunksize